### DISCUSSION TITLE
suggestion: consider the levenshtein distance the search

### DISCUSSION THREAD

**Discussion by c33s** [knqyf263/pet#280] [Ideas]
https://en.wikipedia.org/wiki/Levenshtein_distance

started to open an issue here but i noticed that it might be the wrong location as pet simply uses `fzf`. i opened an issue there but maybe it also can handled in pet

https://github.com/junegunn/fzf/issues/2272

**Comment by c33s** [comment]
as my ticket at junegunn/fzf was closed, it would be awesome for solving it here even it is the wrong location but i am not sure if this is even possible.

**Comment by roeniss** [comment]
I'm not familiar those algorithm neither maintainer of `pet` but imho, looks like we need to find some package (I hope it would be golang-based to make issue simpler) which execute levenshtein search algorithm.

As mentioned in Readme, there are already two available module `fzf` and `peco`. So I think it's not that hard to attach another one, if it exists.

If not, that must be totally different issue. Because that means that we should make new search algorithm implementation at all.

**Comment by RamiAwar** [comment]
Hey @c33s , I think it would be nice to support customizable search algorithms! There's a lot of other issues we need to address first, but if you're interested in working on this you'd be more than welcome :) I can always help but personally need to prioritize other issues. Moving this to #ideas
